weather-ai/src/lib/prompting/index.ts

```diff
@@ -3,20 +3,17 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { ReadableStream } from "stream/web";
 
 export default class BuiltinPrompting {
-    constructor(private session: any) {}
+    constructor(private session: AITextSession) {}
 
-
-    streamingPrompt(prompt: string): any {
-        // Prompt the model and stream the result:
-        const stream = this.session.promptStreaming(prompt);
-        const reader = stream.getReader();
-        return reader;
+    async streamingPrompt(prompt: string): Promise<ReadableStream<string>> {
+        return await this.session.promptStreaming(prompt);
     }
     
     async prompt(prompt: string): Promise<string> {
-        return await this.session.execute(prompt);        
+        return await this.session.prompt(prompt);
```

Comment:
Why the need to `await`? Same above.
```suggestion
        return this.session.prompt(prompt);
````

Reply:
Good catch - I created another PR here, since this one was already merged: https://github.com/GoogleChromeLabs/web-ai-demos/pull/9